README.md

```diff
@@ -24,173 +24,230 @@ Docker images are the preferred way to deploy this code in a production environm
+
+To quickly set up a development environment, we include a [docker-compose-dev.yml](./docker-compose-dev.yml)
+file. This docker-compose file, will build the webapp and scripts containers from your local repository, and start
+those containers as well as all the necessary service containers.
 
-Create a virtual environment with Python 3.11.1, run:
+But first, install docker if not yet installed:
 
-```shell
-make venv
+```sh
+brew install --cask docker
 ```
 
-Activate the virtual environment and install dependencies with Poetry:
+Start the Docker app and then check it's running:
 
-```shell
-pyenv activate circ-311
-poetry install
+```sh
+docker ps
 ```
 
-If all went well, you should see the activated virtual environment `(circ-311)` in your shell and all packages
-installed. You can verify this by checking the Python version:
+Add the url for `ADMIN_EKIRJASTO_AUTHENTICATION_URL` in the `docker-compose-dev.yml` file. Then build the containers by
+running the following command:
 
-```shell
-python3 --version
+```sh
+docker compose -f docker-compose-dev.yml up --build -d
 ```
 
-Now, all you need to do to start docker containers and run the application is:
+There is now a web server listening on port `6500`:
 
-```shell
-make run
+```sh
+curl http://localhost:6500/
 ```
 
-When you need to stop the application and containers and delete everything to start fresh, first check the path of the
-postgres docker container volume in `docker-compose-dev.yml`. Update the `POSTGRES_DATA` in the Makefile to match it
-and then run:
+Check out the [Docker README](/docker/README.md) in the `/docker` directory for in-depth information on running and
+developing the Circulation Manager locally with Docker, or for deploying the Circulation Manager with Docker.
 
-```shell
-make clean
-```
+For setting up a library with collections, you can skip over to section [The Admin Interface](#admin-interface).
 
-You can modify the Makefile to also create a virtual enviroment for Python 3.10.1. You can switch between the virtual
-environments by activating and deactivating them with e.g. `pyenv activate circ-311` and `pyenv deactivate circ-311`.
+### 2. Python and dependencies to run tests (and the application locally)
 
-### Docker Compose
+#### Python Setup
 
-In order to help quickly set up a development environment, we include a [docker-compose.yml](./docker-compose.yml)
-file. This docker-compose file, will build the webapp and scripts containers from your local repository, and start
-those containers as well as all the necessary service containers.
+In order to run tests or the code outside Docker, you'll need to set up Python and a virtual environment. Macs should
+have Python already installed.
 
-You can give this a try by running the following command:
+#### Dependencies
 
-```shell
-docker-compose up --build
+Install the following required dependencies:
+
+```sh
+brew install pkg-config libffi
+brew install tvuotila/libxmlsec1/libxmlsec1@1.2.37
```

Comment:
Could not be installed. Homebrew displayed this error when trying to install `Error: openssl@1.1 has been disabled because it is not supported upstream! It was disabled on 2024-10-24.`